packages/client/libclient-py/quickmpc/request/qmpc_request.py

```diff
@@ -160,9 +160,14 @@ def send_share(self, df: pd.DataFrame, piece_size: int = 1_000_000) \
                 "piece_size must be in the range of 1000 to 1000000")
         if df.isnull().values.sum() != 0:
             raise RuntimeError("規定されたフォーマットでないデータです．")
+        sort_index_name = "__qmpc_sort_index__"
+        if sort_index_name not in df.columns:
+            raise RuntimeError("規定されたフォーマットでないデータです．"
+                               f"columnsに{sort_index_name}が含まれてる必要があります．")
```

Comment:
@mdonaka 
Is there documentation available that explains `__qmpc_sort_index__`?
When users see this error message, they may ask, "What is `__qmpc_sort_index__`?"
I think it would be a good idea to provide a link to a document that would be helpful!

Reply:
I add a doc! [fed12fc](https://github.com/acompany-develop/QuickMPC/pull/255/commits/fed12fc93b1df8a9208ad8fa09925739e715f0d6)
Other docs are in preparation and will not be added here.